src/Paginator.php

```diff
@@ -894,7 +893,7 @@ protected function _getCacheId($page = null)
     protected function _getCacheInternalId()
     {
         $adapter            = $this->getAdapter();
-        $adapterToSerialize = $adapter instanceof DbSelect
+        $adapterToSerialize = method_exists($adapter, 'getArrayCopy')
```

Comment:
This error should probably be added to `phpstan-baseline.xml`

Reply:
you mean psalm-baseline.xml ?

Reply:
I've updated baseline using this command:
`./vendor/bin/psalm --update-baseline --set-baseline=psalm-baseline.xml`
Not sure about what I'm doing ^^"

Reply:
`--update-baseline` removes 'fixed' issues from the baseline

`--set-baseline=psalm-baseline.xml` adds new, un-fixed issues to the baseline and also updates it at the same time.

Reply:
Current diff is OK BTW :+1: